packages/manager/apps/pci-block-storage/public/translations/retype/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_storages_blocks_retype_title": "Modifier votre type d'offre",
+  "pci_projects_project_storages_blocks_retype_subtitle": "Lorem ipsum dolor sit amet consectetur. Hac natoque fames hac viverra bibendum sodales nisl.",
```

Comment:
lorem ipsum ?

Reply:
Yay i know, the figma has a subtitle but it's "lorem ipsum" so far so I figure I might as well add it and change it with the correct version afterwards